techmatters/flex-plugins
# fix: fetch case when not loaded into state [CHI-2456]

## Description


### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-2456)
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Verification steps
- Edit `src/public/appConfig.js` like `const accountSid = 'ACxxxxxxxxxxxxxxxxxx'; // as staging` where `ACxxxxxxxxxxxxxxxxxx` is the account sid of Aselo staging.
- Follow the reproduction steps of the linked Jira ticket and confirm you can't replicate those bugs anymore.
